src/xenia/base/vec128.h

```diff
@@ -90,7 +90,8 @@ typedef struct alignas(16) vec128_s {
       uint32_t uz;
       uint32_t uw;
     };
-    float f32[4];
+	float f32[4];
+	double f64[2];
```

Comment:
On the SSE and AVX stuff. I noticed Halo Reach and Halo 3 use this code a lot and when I debugged it appeared to be positional co-ordinates, thats why I was eager to smash out some faster code for it